include/mujincontrollerclient/mujinjson.h

```diff
@@ -37,6 +37,15 @@
 #include <rapidjson/error/en.h>
 #include <rapidjson/prettywriter.h>
 
+#ifndef MUJINCLIENTJSON_LOAD_REQUIRED_JSON_VALUE_BY_KEY
+#define MUJINCLIENTJSON_LOAD_REQUIRED_JSON_VALUE_BY_KEY(rValue, key, param) \
```

Comment:
Is there a reason this is a macro instead of a function `LoadRequiredJsonValueByKey`?

Reply:
ya, actually I also think function is better, then we can put it in a namespace...

Reply:
should we talk in controllercommon!489 ?

Reply:
I see, Ziyan brought up the same concern [here](https://git.mujin.co.jp/dev/controllercommon/-/merge_requests/489#note_313010). Let's keep discussion here for now.

Reply:
That makes you, me and @ziyan arguing for a function over a macro then